http/server.go

```diff
@@ -313,3 +314,29 @@ func (h *handler) ChainInfo(w http.ResponseWriter, r *http.Request) {
 	http.ServeContent(w, r, "info.json", time.Unix(info.GenesisTime, 0), bytes.NewReader(chainBuff.Bytes()))
 
 }
+
+func (h *handler) Health(w http.ResponseWriter, r *http.Request) {
+	h.pendingLk.RLock()
+	lastSeen := h.latestRound
+	h.pendingLk.RUnlock()
+
+	info := h.getChainInfo(r.Context())
+	if info == nil {
+		w.Header().Set("Content-Length", "11")
```

Comment:
Isn't his auto-set by Go?

Reply:
if we explicitly call `WriteHeader` to set the status code, then we need to set it